server/@types/ui/index.d.ts

```diff
@@ -501,7 +502,8 @@ export type SpaceSearchFormData = SpaceSearchCommonFields & {
   apType?: ApTypeCriteria
   newPlacementArrivalDate?: string
   newPlacementDepartureDate?: string
-  newPlacementReason?: string
+  newPlacementReason?: TransferReason
+  notes?: string
```

Comment:
I would possibly suggest prefixing this with `newPlacement` -- this was done to all other fields originally to differentiate the session properties solely used in the new placement flow. There might be a better way to handle this, maybe a separate level for new placement? Not sure this is desirable, but not sure why I decided against it in the first place either 😕